Header header

string[] bag_profile_names
bool[] bagging
